container_checkpoint_test.go

```diff
@@ -47,7 +47,7 @@ func TestCheckpointRestorePTY(t *testing.T) {
 		t.Fatal(err)
 	}
 	defer client.Close()
-	if client.runtime == v1runtime {
+	if client.runtime == "io.containerd.runtime.v1.linux" {
```

Comment:
Can we use constant `github.com/containerd/containerd/plugin.RuntimePlugin`? (If we don't want to import `plugin` package can we split the consts from `plugin`?)